easybuild/easyblocks/generic/rpackage.py

```diff
@@ -215,11 +218,22 @@ def run(self):
             lib_install_prefix = os.path.join(self.installdir, self.cfg['exts_subdir'])
             mkdir(lib_install_prefix, parents=True)
 
-        if self.patches:
+        if self.patches or self.cfg['update_config_guess']:
             super(RPackage, self).run(unpack_src=True)
         else:
             super(RPackage, self).run()
 
+        if self.cfg['update_config_guess']:
+            confmake = ConfigureMake(self.cfg)
+            for root, _, files in os.walk(self.builddir):
+                for name in files:
+                    if name == 'config.guess':
+                        confmake.config_guess = os.path.join(root, name)
+                        if not confmake.check_config_guess():
```

Comment:
The ConfigureMake EB does this unconditionally (probably based on the assumption that it is highly unlikely that the config.guess matches). So I'd do the same here.

As an alternative: Add a quiet option to the check_config_guess to avoid the warnings. IMO there is nothing to warn about here. For the ConfigureMake EB the updating of the config.guess is the natural thing to do. So the output generated by `obtain_config_guess` is enough